src/components/NotificationsDrawer/NotificationItem.tsx

```diff
@@ -46,7 +46,10 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ notification, onNav
 
   const notificationDropdownItems = [
     <DropdownItem key="read" onClick={onMarkAsRead}>{`Mark as ${!notification.read ? 'read' : 'unread'}`}</DropdownItem>,
-    <DropdownItem key="manage-event" onClick={() => onNavigateTo('settings/notifications/configure-events')}>
+    <DropdownItem
+      key="manage-event"
+      onClick={() => onNavigateTo(`settings/notifications/configure-events?bundle=${notification.bundle}&tab=configuration`)}
```

Comment:
The URL has to be absolute. If you click the item from a nested router, the pathname will append and you will end up on non existing page.

Reply:
fixed